app.py

```diff
@@ -19,16 +16,46 @@ def get(self):
 class AjaxEchoHandler(tornado.web.RequestHandler):
     def get(self):
         count =self.get_argument('count')
-        data = {'count': int(count) + 1}
+        body =self.get_argument('body')
+        data = {'count': int(count) + 1, 'body':body}
         self.write(data)
 
+class WebsocketTestHandler(tornado.web.RequestHandler):
+    def get(self):
+        self.render('websockettest.html')
+
+class WebsocketTalkHandler(tornado.websocket.WebSocketHandler):
+    waiters = []
+
+    def get_compression_options(self):
+        return {}
+
+    def open(self):
+        WebsocketTalkHandler.waiters.append(self)
+
+    def on_close(self):
+        WebsocketTalkHandler.waiters.remove(self)
+
+    @classmethod
+    def send_updates(cls, message):
+        #logging.info("sending message to %d waiters", len(cls.waiters))
+        cls.waiters[0].write_message(message)
```

Comment:
http://www.tornadoweb.org/en/stable/websocket.html#tornado.websocket.WebSocketHandler.get_compression_options

for the compression. oh, I see it would not be fair :)